bundle/camunda-saas-bundle/src/main/java/io/camunda/connector/runtime/saas/CamundaClientSaaSConfiguration.java

```diff
@@ -68,14 +69,16 @@ public CamundaClientConfiguration saasCamundaClientConfiguration(
       final JsonMapper jsonMapper,
       final List<ClientInterceptor> interceptors,
       final List<AsyncExecChainHandler> chainHandlers,
-      final CamundaClientExecutorService zeebeClientExecutorService) {
+      final CamundaClientExecutorService zeebeClientExecutorService,
+      final JobExceptionHandlerSupplier jobExceptionHandlerSupplier) {
```

Comment:
Why do you create your own configuration instance?

Reply:
Here we go: 
https://github.com/camunda/connectors/issues/5710